nuxt.config.ts

```diff
@@ -161,6 +161,11 @@ export default defineNuxtConfig({
         path: "/product/:productCode/",
         component: resolve(__dirname, "pages/product/index.vue"),
       })
+      routes.push({
```

Comment:
remove dev route

Reply:
This route has been created to handle product-zoom page route